api/src/main/java/io/minio/messages/ErrorResponse.java

```diff
@@ -64,6 +64,15 @@ public ErrorResponse(Reader reader) throws IOException, XmlPullParserException {
     this.parseXml(reader);
   }
 
+  /**
+   * Constructs a new ErrorResponse object in case of non-xml response.
+   */
+  public ErrorResponse(String message) throws IOException, XmlPullParserException {
```

Comment:
This constructor is not used and not required.

Reply:
Removed.